python/ray/tune/tune.py

```diff
@@ -14,33 +14,16 @@
 from ray.tune.syncer import wait_for_sync, set_sync_periods, SyncConfig
 from ray.tune.trial_runner import TrialRunner
 from ray.tune.progress_reporter import CLIReporter, JupyterNotebookReporter
-from ray.tune.schedulers import (HyperBandScheduler, AsyncHyperBandScheduler,
-                                 FIFOScheduler, MedianStoppingRule)
 
 logger = logging.getLogger(__name__)
 
-_SCHEDULERS = {
-    "FIFO": FIFOScheduler,
-    "MedianStopping": MedianStoppingRule,
-    "HyperBand": HyperBandScheduler,
-    "AsyncHyperBand": AsyncHyperBandScheduler,
```

Comment:
Hey @ttumiel Just a nit - can you make sure this backwards compatibility isn't broken?

For `FIFO`, I think it's safe just to return `FIFOScheduler()` as a special case.

Reply:
(it's also safe to duplicate entries in the SCHEDULER_IMPORT dict

Reply:
Just to be sure - do you mean that I shouldn't remove `_SCHEDULERS` and `_make_scheduler`?

Reply:
I _think_ he was suggesting the following:
- remove `_SCHEDULERS` and `_make_scheduler`
- update the keys in `SCHEDULER_IMPORT` with the keys from `_SCHEDULERS`

so `SCHEDULER_IMPORT` would look kind of like this:
```python
SCHEDULER_IMPORT = {
    "async_hyperband": AsyncHyperBandScheduler,
    "median_stopping_rule": MedianStoppingRule,
    "hyperband": HyperBandScheduler,
    "hb_bohb": HyperBandForBOHB,
    "pbt": PopulationBasedTraining,
    # from _SCHEDULERS:
    "FIFO": FIFOScheduler,
    "MedianStopping": MedianStoppingRule,
    "HyperBand": HyperBandScheduler,
    "AsyncHyperBand": AsyncHyperBandScheduler,
}
```

but @richardliaw do we even need backwards compatibility, since `_make_scheduler` is internal? If we just update all the previous `_make_scheduler` calls to use the snake_case keys then we can keep the current `SCHEDULER_IMPORT`, right?

Reply:
hmm its exposed in rllib afaik, so for now lets keep the compatibility!

Reply:
Ah I see! Will do